src/extension.ts

```diff
@@ -184,13 +184,14 @@ async function init(
 
 	await vscode.commands.executeCommand('setContext', 'github:initialized', true);
 
+	const experimentationService = await createExperimentationService(context, telemetry);
+	await experimentationService.initializePromise;
+	experimentationService.isCachedFlightEnabled('githubaa');
```

Comment:
This method call looks like it returns something that should be read. I guess it also has side effects though and the return value doesn't matter?

Reply:
Yeah, that's right - normally we would use the return value to turn a feature on or off, but doing this also updates info about what flight a user is in so just tests that everything is set up correctly